gbm_drm_gen/drmgen.py

```diff
@@ -152,10 +153,41 @@ def set_location(self, ra, dec):
             self._drm = self._make_drm(az, el, self._geo_az, self._geo_el)
 
         # go ahead and transpose it for spectal fitting, etc.
-        #self._drm_transpose = self._drm.T
+        # self._drm_transpose = self._drm.T
+
+    def set_location_direct_sat_coord(self, ra, dec):
```

Comment:
do you mean RA ad DEC in satellite coordinates? This should be named to Az and Alt